CHANGELOG.adoc

```diff
@@ -56,6 +56,7 @@ It will start by improving a bit performances since diagrams were persisted twic
 - https://github.com/eclipse-sirius/sirius-components/issues/2008[#2008] [view] Make edge tool more robust for applications which use a custom IViewRepresentationDescriptionSearchService
 - https://github.com/eclipse-sirius/sirius-components/issues/2027[#2027] [diagram] Support both versions of the ITool interface in the imageURL data fetchers
 - https://github.com/eclipse-sirius/sirius-components/issues/2026[#2026] [view] Display the layout strategy description of a node description in the explorer view again
+- https://github.com/eclipse-sirius/sirius-components/issues/2032[#2032] [form] Add domainType on PageDescription in the view DSL and takes into account the impacts for the FormDescriptionAggregator.
```

Comment:
Could you add some information for the other changes such as the breaking change to contribute the to `IPropertiesDescriptionRegistry` which takes a PageDescription and not a FormDescription anymore and the new selection variable available in some contexts